crimsobot/cogs/mystery.py

```diff
@@ -44,8 +65,8 @@ async def ppf(self, ctx: commands.Context, spread: str = 'ppf') -> None:
             footer='Type ">tarot card" for more on a specific card.',
         )
 
-        for card_tuple in descriptions:
-            embed.add_field(name=card_tuple[0], value='**{}**\n{}'.format(card_tuple[1], card_tuple[2]))
+        card_tuple = descriptions[0]
+        embed.add_field(name=card_tuple[0], value='**{}**\n{}'.format(card_tuple[1], card_tuple[2]))
```

Comment:
looks like https://github.com/crimsobot/crimsoBOT/commit/328d54e0051d34878dfc897987c60f158e6f150c fixed the wrong method